src/M6Web/Bundle/LogBridgeBundle/Config/FilterCollection.php

```diff
@@ -130,17 +122,15 @@ public function getKeys()
      *
      * @return Filter
      */
-    public function current()
+    public function current(): mixed
```

Comment:
**suggestion (non-blocking):**

```suggestion
    public function current(): Filter
```

🤷

Reply:
it seems that the `$this->get(...)` function is able to return `null`

so this would be more accurate:
```suggestion
    public function current(): ?Filter
```

this could happen when the FilterCollection is empty, maybe? 🤷 
(I have no idea what this class is about...)

Reply:
Yep, it returns `null` if the collection is empty, I changed the code accordingly